preprocessing/structs.ts

```diff
@@ -118,6 +118,9 @@ import "mgv_lib/LogPriceConversionLib.sol";
 using OfferPackedExtra for OfferPacked global;
 using OfferUnpackedExtra for OfferUnpacked global;
 
+// cleanup-mask: 0s at location of fields to hide from maker, 1s elsewhere
+uint constant hide_fields_from_maker_mask = ~(prev_mask_inv | next_mask_inv);
```

Comment:
```suggestion
uint constant HIDE_FIELDS_FROM_MAKER_MASK = ~(prev_mask_inv | next_mask_inv);
```

Reply:
Addressed in 20bcff1575e55fdf83fbd3b570d066cfb589d250